src/universal/components/Action/Action.js

```diff
@@ -37,6 +38,9 @@ const Action = (props) => {
         }
         <AsyncRoute isAbstract isPrivate path="(/me|/newteam|/team)" mod={dashWrapper} />
         <AsyncRoute isPrivate path="/meeting/:teamId/:localPhase?/:localPhaseItem?" mod={meetingRoot} />
+        {__RELEASE_FLAGS__.newSignIn &&
```

Comment:
should we use a new flag? I imagine we'll finish the sign in long before we finish the retros.

Reply:
You're totally right. I was in auto-pilot mode here!